### PR TITLE
Fixed errors in development guidelines

### PR DESCRIPTION
**Pull request by michizhou**
Corrected some grammatical mistakes and typos in documentation for development guidelines, as well as fixing capitalization errors to ensure consistency of terms.

### CODE REVIEW

File: Development-Guidelines/test-driven-flow.md [hunk 1]
```diff
@@ -78,21 +78,21 @@ So to add a service, which requires data from the server we should:
...
-There is a static servervariables file in /mocks which describes the urls used by the rest service, this is currently needed as we don't have this set as an angular service, and no real conventions for these urls yet. Longer-term it would be great to have a urlBuilder which could do
+There is a static servervariables file in /mocks which describes the urls used by the rest service, this is currently needed as we don't have this set as an Angular service, and no real conventions for these urls yet. Longer-term it would be great to have a urlBuilder which could do
 
 ```javascript
 urlService.url("contentTypes", "GetAllowedChildren");
 // would return /<umbracodir>/<apibaseDir>/contentyTypes/getAllowedChildren
 ```
 
-But for now, they are set in the servervariables file.
+But for now, they are set in the server variables file.
```
Comment (sofietoft) [inline Development-Guidelines/test-driven-flow.md]: I'm fairly certain this is actually a contept in Umbraco, so let's keep this as one word!
I think we should even capitalize it like "ServerVariables" 🤔

**Comment by sofietoft** [comment]
Brilliant @michizhou ! Thanks 😎 🎉

**Comment by michizhou** [comment]
@sofietoft Sorry for not bringing this up earlier, but I noticed you never added the special  “Hacktoberfest 2019 Contributor” label to my PR. As a result, I was never notified about claiming swag for my contribution. If it's still possible, I'd appreciate it if you could send me some Umbraco swag (stickers and/or otherwise) in recognition of my contribution.

**Comment by sofietoft** [comment]
Hi @michizhou 
Unfortunately your contribution here doesn't meet the [quality standard guidelines for Umbraco Hacktoberfest swag](https://umbraco.com/blog/hacktoberfest-2019/) - which also follows the Quality Standards for the official Hacktoberfest: https://hacktoberfest.digitalocean.com/details#quality-standards

**Comment by michizhou** [comment]
@sofietoft I fail to understand why my contribution here fails to meet the quality standards for Umbraco Hacktoberfest swag when a PR like #2056 meets the standards.

**Comment by dampee** [comment]
@michizhou I understand your feeling very well. 

What @sofietoft tried to explain that we were not looking for simply typo fixes for the docs, as explained in the Umbraco blog post about hacktober fest:
"In general, we encourage all kinds of contributions, whether it’s whole new articles in the Documentation or bug fixes in the CMS."

Although I am sure @nul800sebastiaan had good reasons to mark the other PR (#2056) with the hacktober label. It does not to be fair right now.

Let's take a deep breath, and wait for their answers.

**Comment by sofietoft** [comment]
That's a fair point @michizhou !
Looks like I was a bit fast to judge here. Sorry about that!
We haven't started sending out any swag yet, so I should be able to get you on the list in time.

**Comment by nul800sebastiaan** [comment]
Hi there @michizhou,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing 🥇 

We have had another record breaking amount of PR's in both the CMS, Documentation, and other smaller public repositories. You can read all about the numbers in the [Umbraco Hacktoberfest round-up blog post](https://umbraco.com/blog/pr-team-update-november-2019-hacktoberfest-round-up/).

Due to the amazing work you and others in the community have been doing, we've had a bit of a hard time keeping up. 😅 While all of the PRs for Hacktoberfest might not have been merged yet, you still qualify for receiving some Umbraco swag, congratulations! 🎉 

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://docs.google.com/forms/d/e/1FAIpQLSc-K97Zl5k86cVebKWWqieLM4Fm_jCMGVdbQdYnq4IUJ7EvZg/viewform) :point_left: Please follow this link to fill out and submit the form, **before December 19th 2019**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before February, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**